.jenkins

```diff
@@ -0,0 +1,114 @@
+pipeline {
+  agent none
+
+  options {
+    disableConcurrentBuilds(abortPrevious: true)
+    timeout(time: 6, unit: 'HOURS')
+  }
+
+  triggers {
+    issueCommentTrigger('.*test this please.*')
+  }
+
+  stages {
+    stage('Build') {
+      parallel {
+        stage('CUDA-12.6-NVCC') {
+          agent {
+              dockerfile {
+                  filename 'Dockerfile.nvcc'
+                  dir 'scripts'
+                  label 'nvidia-docker'
+                  args '--env NVIDIA_VISIBLE_DEVICES=$NVIDIA_VISIBLE_DEVICES'
+              }
+          }
+          steps {
+            sh '''rm -rf kokkos &&
+                  git clone --branch 4.4.01 --single-branch https://github.com/kokkos/kokkos.git && \
```

Comment:
```suggestion
                  curl -sLo https://github.com/kokkos/kokkos/releases/download/4.4.01/kokkos-4.4.01.tar.gz && \
```

Reply:
And `tar -xz ...`

Reply:
I just used `--depth 1`, so I don't have to untar.

Reply:
ok